plugins/janus_sip.c

```diff
@@ -1856,9 +1856,10 @@ static void janus_sip_hangup_media_internal(janus_plugin_session *handle) {
 	session->media.simulcast_ssrc = 0;
 	if(!(session->status == janus_sip_call_status_inviting ||
 		 session->status == janus_sip_call_status_invited ||
-		 session->status == janus_sip_call_status_incall))
+		 session->status == janus_sip_call_status_incall)) {
```

Comment:
Very good catch! And I noticed we have the exact same issue in the SIPre plugin (and maybe others?) so I'll make sure to apply the same fix there.

Reply:
I focus mainly on sip plugin. Anyway , you guys did a great work.

Reply:
Thanks for the kind words!